tinkerers/Afra A.md

```diff
@@ -0,0 +1,9 @@
+#**Afra A**
+
+
+Hi there, I am Afra
+I am currently pursuing Btech in Artificial Intelligence and Data Science at Ilahia College of Engineering and Technology, Muvattupuzha. I love to program and create something.
+
+#**My Socials**
+
+Linkedin :https://www.linkedin.com/in/afra-a-20808a233
```

Comment:
Heyy Afra, please use proper markdown syntax to display this URL as a hyperlink.

```
like this 👇
[Linkedin](YOUR_PROFILE_URL)
```